pytorch/training/docker/1.12/py3/cu113/Dockerfile.gpu

```diff
@@ -515,6 +515,16 @@ RUN  wget -nv https://github.com/rapidsai/rmm/archive/v${RMM_VERSION}.tar.gz \
 # Install ZeRO-2D Binary
 RUN pip install --no-cache-dir -U ${ZERO_2D_URL}
 
+# Install flash attention
```

Comment:
Why is this needed? do we have test to cover this new package?

Reply:
Yes this is part of an S team goal for the release. It's a dependency of SMP going forward. This test touches that, but for proper test we'll need p4d. Running below test  for p4d will run with flash attention

https://github.com/aws/deep-learning-containers/blob/master/test/sagemaker_tests/pytorch/training/integration/sagemaker/test_distributed_operations.py#L205

Reply:
is the test configured with p4d? I see the instance_type as `p3.16xlarge`

Reply:
Please make changes to the test configuration in order to test this functionality.

Reply:
Yes, I chatted with Shibo offline that it should be `p4d.24xlarge` instead, thanks.